components/menu/index.css

```diff
@@ -465,7 +487,7 @@
 
 	&:focus,
 	&.is-focused {
-		background-color: var(--highcontrast-menu-item-background-color-focus, var(--mod-menu-item-background-color-key-focus, var(--spectrum-menu-item-background-color-key-focus)));
+		background-color: var(--highcontrast-menu-item-background-color-focus, var(--mod-menu-item-background-color-key-focus, var(--spectrum-menu-item-background-color-keyboard-focus)));
```

Comment:
I noticed we're missing a background color on the keyboard focused menu item and I'm wondering if the `:focus` and `:focus-visible` states have been being confused for one another? That might need a second look.

I think `.is-focus-visible` is being applied in the template for the `isFocused` arg, which feels like it also might be something we've been getting wrong for awhile unless I'm missing something?

One other issue I noticed related to this is that the focused menu items in the testing grid aren't getting the right content colors either (they get the default color instead of the focus color), because I think they might be applied specifically only to `:focus` and not `:focus-visible`.

![image](https://github.com/user-attachments/assets/93384133-22d4-48dc-8c0d-34fb2d6f8299)